index.html

```diff
@@ -38,5 +38,161 @@ <h2>
+              <h1>Profesional Art Printing Data</h1>
+              <p>
+                  A daily selection of privately personalized reads; 
+                  no accounts or sign-ups required. 
+                  has been the industry's standard
+                </p>
+                <ul>
+                  <li class="html">html</li>
+                  <li class="boostrap">boostrap</li>
+                  <li class="ruby">ruby</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="card-works">
+              <h1>Profesional Art Printing Data</h1>
+              <p>
+                  A daily selection of privately personalized reads; 
+                  no accounts or sign-ups required. 
+                  has been the industry's standard
+                </p>
+                <ul>
+                  <li class="html">html</li>
+                  <li class="boostrap">boostrap</li>
+                  <li class="ruby">ruby</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="card-works">
+              <h1>Profesional Art Printing Data</h1>
+              <p>
+                  A daily selection of privately personalized reads; 
+                  no accounts or sign-ups required. 
+                  has been the industry's standard
+                </p>
+                <ul>
+                  <li class="html">html</li>
+                  <li class="boostrap">boostrap</li>
+                  <li class="ruby">ruby</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="card-works">
+              <h1>Profesional Art Printing Data</h1>
+              <p>
+                  A daily selection of privately personalized reads; 
+                  no accounts or sign-ups required. 
+                  has been the industry's standard
+                </p>
+                <ul>
+                  <li class="html">html</li>
+                  <li class="boostrap">boostrap</li>
+                  <li class="ruby">ruby</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="card-works">
+              <h1>Profesional Art Printing Data</h1>
+              <p>
+                  A daily selection of privately personalized reads; 
+                  no accounts or sign-ups required. 
+                  has been the industry's standard
+                </p>
+                <ul>
+                  <li class="html">html</li>
+                  <li class="boostrap">boostrap</li>
+                  <li class="ruby">ruby</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+            <div class="about-me">
+              <h1>About me</h1>
+              <p>
+                  Hello I’m a software developer! I can help you build a product , 
```

Comment:
- [ ] As mentioned by the previous reviewer, Kindly consider using the semantic tags for accessibility purposes on your page over the internet.
` Work section is supposed to be inside a <section> tag and about in another one. Not all combined in one.`